docs/src/componentDocs/DrawerNavItem/examples/SelectingANavItem.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {Box} from '@mui/material';
+import {SelectingANavItemExample} from './SelectingANavItemExample';
+import {CodeBlock, CodeBlockActionButtonRow} from '../../../shared';
+
+const codeSnippet = `<Drawer activeItem="item2">
+    <DrawerBody>
+        <DrawerNavItem
+            itemID={'item1'}
+            title={'Account'}
+            icon={{family: 'material-community', name: 'account', direction: 'auto'}}
+        />
+        <DrawerNavItem
+            itemID={'item2'}
```

Comment:
Can you show round activeItemBackgroundShape? Check https://brightlayer-ui-components.github.io/react-dev/components/drawer-nav-item/examples